pandas/_libs/missing.pyx

```diff
@@ -16,17 +17,12 @@ from pandas._libs.tslibs.nattype cimport (
     checknull_with_nat, c_NaT as NaT, is_null_datetimelike)
 from pandas._libs.ops_dispatch import maybe_dispatch_ufunc_to_dunder_op
 
-from pandas.compat import is_platform_32bit
-
-
 cdef:
     float64_t INF = <float64_t>np.inf
     float64_t NEGINF = -INF
 
     int64_t NPY_NAT = util.get_nat()
 
-    bint is_32bit = is_platform_32bit()
```

Comment:
can you do `bint is_32bit = platform.architecture()[0] == "32-bit"` and avoid re-computing at runtime?